pkgs/applications/graphics/ahoviewer/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, pkgs, fetchurl, fetchFromGitHub, pkgconfig, libconfig, gtkmm, glibmm, libxml2, gstreamer, libsecret, curl, unrar, libzip, autoconf, automake }:
+with stdenv.lib;
+stdenv.mkDerivation rec {
+  name = "ahoviewer-1.4.6";
+  rev = "414cb91d66d96fab4b48593a7ef4d9ad461306aa";
+  src = fetchFromGitHub {
+    owner = "ahodesuka";
+    repo = "ahoviewer";
+    inherit rev;
+    sha256 = "081jgfmbwf2av0cn229cf4qyv6ha80ridymsgwq45124b78y2bmb";
+  };
+  buildInputs = [ pkgconfig glibmm libconfig gtkmm glibmm libxml2 gstreamer libsecret curl unrar libzip autoconf automake ];
```

Comment:
If you want it to be 100% correct add `pkgconfig` `autoconf` `automake` to `nativeBuildInputs`. `buildInputs` are dependencies used by the package while `nativeBuildInputs` are needed to build the package. This is useful for cross builds (64bit host builds 32bit binaries) where `nativeBuildInputs` need to match the hosts architecture and `buildInputs` the one of the build target.